amplipi/models.py

```diff
@@ -1375,3 +1392,423 @@ class Config:
         }
       ]
     }
+
+
+class PlayItemResponse(BaseModel):
+  directory: Optional[str]  # Directory that the browser is in
+  status: Status
```

Comment:
jw - why do we return status here? it's huge

Reply:
context: it's because that was the prior interface for this type of response, and we're retaining backwards compatibility